ocaml/xapi/xapi_guest_agent.ml

```diff
@@ -138,7 +138,7 @@ let other all_control =
     the results of these lookups differ *)
 
 type m = (string * string) list
-let cache : (int, (m*m*m*m*m*m*bool*bool*bool*float)) Hashtbl.t = Hashtbl.create 20
+let cache : (int, (m*m*m*m*m*m*bool*bool*bool*float*API.tristate_type*API.tristate_type)) Hashtbl.t = Hashtbl.create 20
```

Comment:
One of those things that's grown little by little until it's nasty...
I'll add a refactoring commit to fix it.